### PR TITLE
Content changes

### PR DESCRIPTION
**Pull request by syed87**
6 Content changes made the details are in the ticket below

https://trello.com/c/82nfMggn/2352-content-changes-ticket

1. Request LoPS Verification Page Text Update
2. Typo on Verify LoPS Page
3. Typos on Request LoPS Verification Page
4. Color of Review Tag
5. Admin Journey - Task Status: Accepting LoPS
6. Errors on the ‘review verifications’ screen:

### CODE REVIEW

File: app/views/assessor_interface/review_verifications/index.html.erb [hunk 1]
```diff
@@ -3,6 +3,10 @@
 
 <h1 class="govuk-heading-xl">Review verifications</h1>
 
+<p class="govuk-body">
+  The following verification tasks ahve been flagged for review:
```
Comment (thomasleese) [inline app/views/assessor_interface/review_verifications/index.html.erb:7]:
```suggestion
  The following verification tasks have been flagged for review:
```

File: app/views/assessor_interface/professional_standing_requests/show.html.erb [hunk 2]
```diff
@@ -4,7 +4,7 @@
 <h1 class="govuk-heading-xl">Verify LoPS</h1>
 
 <p class="govuk-body">
-  You need to request verification for this applications LoPS. Follow the steps below:
+  You need to request verification for this application's LoPS.
```
Comment (thomasleese) [inline app/views/assessor_interface/professional_standing_requests/show.html.erb:7]:
```suggestion
  You need to request verification for this application’s LoPS.
```

**Review by thomasleese — COMMENTED** [review]
Just a couple of small comments, but looks good. Once this is merged in to `verification-lops` I'll squash it in to the commits there.

File: app/views/assessor_interface/professional_standing_requests/show.html.erb (hunk 2 above)
Comment (syed87) [reply app/views/assessor_interface/professional_standing_requests/show.html.erb:7]: I can't see the difference for this change

Comment (thomasleese) [reply app/views/assessor_interface/professional_standing_requests/show.html.erb:7]: It's a different kind of apostrophe, it just looks nicer: https://cmosshoptalk.com/2019/03/19/smart-apostrophes-cmos-6-117/

File: app/views/assessor_interface/professional_standing_requests/edit_request.html.erb [hunk 3]
```diff
@@ -5,18 +5,18 @@
 
 <h1 class="govuk-heading-xl"><%= title %></h1>
 
-<p class="govuk-body">To verify the applications LoPS you need to:</p>
+<p class="govuk-body">To verify the application's LoPS you need to:</p>
```
Comment (thomasleese) [inline app/views/assessor_interface/professional_standing_requests/edit_request.html.erb:8]:
```suggestion
<p class="govuk-body">To verify the application’s LoPS you need to:</p>
```

**Review by thomasleese — APPROVED** [review]

**Pull request merged**